Lesson1/Dima_Gnatyk/main.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <vector>
+#include <list>
+#include <forward_list>
+#include <numeric>
+
+template <class T>
+void print(const std::vector<T> &data){
+    for(auto value:data){
+        std::cout <<value<<" ";
+    }
+    std::cout<<std::endl;
+}
+
+template <class T>
+std::vector<T> concat(const std::vector<T> &data_vector_1, const std::vector<T> &data_vector_2){
+    if(data_vector_1.empty()) return data_vector_2;
```

Comment:
Please use the figure brackets even if it is a one line statement.
Consider:
if (data_vector_1.empty())
{
    return ....;
}